ush/forecast_predet.sh

```diff
@@ -23,6 +23,87 @@ common_predet(){
 }
 
 DATM_predet(){
+  echo "SUB ${FUNCNAME[0]}: Defining variables for datm"
```

Comment:
How much of this is duplicate code of `FV3_GFS_predet()` that could be abstracted out into a separate function that is called for both cases?